kinesis/src/main/scala/org/apache/pekko/stream/connectors/kinesis/impl/KinesisSchedulerSourceStage.scala

```diff
@@ -49,7 +49,7 @@ private[kinesis] object KinesisSchedulerSourceStage {
 @InternalApi
 private[kinesis] class KinesisSchedulerSourceStage(
     settings: KinesisSchedulerSourceSettings,
-    schedulerBuilder: ShardRecordProcessorFactory => Scheduler)(implicit ec: ExecutionContext)
```

Comment:
Doesn't the existing code pass the materialzer executionContext as the implicit value anyway?

Reply:
yes, but I think the `GraphStageLogic` materializer is different as it takes into consideration the `initialAttributes: Attributes`

Reply:
So I added `println(Thread.currentThread().getName)` to the `Future(scheduler.run())` like this:

```scala
      Future{
        println(Thread.currentThread().getName)
        scheduler.run()
      }
```

and ran the tests and I see `KinesisTests-pekko.actor.default-dispatcher-6` being used

so this PR makes no difference...

If I add the `println` to the `onPull` method I see `KinesisTests-pekko.actor.default-blocking-io-dispatcher-10` being used...